src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -71,5 +71,12 @@ public static class KnownQueryParameterNames
         public const string OutputFormat = "_outputFormat";
 
         public const string TypeFilter = "_typeFilter";
+
+        /// <summary>
+        /// The date range parameters for $everything operation
+        /// </summary>
+        public const string Start = "start";
```

Comment:
>Start [](http://example.com/codeflow?start=28&length=5)

I would keep it separatly from this list.
With exception of Anonymization all that parameters are global or search specific.

maybe something like:
[https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.Core/Features/Operations/ConvertData/Models/ConvertDataProperties.cs](https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.Core/Features/Operations/ConvertData/Models/ConvertDataProperties.cs)

Reply:
Sure, `EverythingOperationParameterNames` is added to hold these parameters.